project/plugins.sbt

```diff
@@ -14,6 +14,6 @@ sys.env.get("PLAY_VERSION") match {
 
 sys.env.get("PLAY_VERSION") match {
   case Some("2.8") => addSbtPlugin("com.typesafe.sbt"        % "sbt-twirl"  % "1.5.1")
-  case Some("2.9") => addSbtPlugin("com.typesafe.play"       % "sbt-twirl"  % "1.6.1")
-  case _           => addSbtPlugin("org.playframework.twirl" % "sbt-twirl"  % "2.0.1")
+  case Some("2.9") => addSbtPlugin("com.typesafe.play"       % "sbt-twirl"  % "1.6.7")
```

Comment:
hmm, I grabbed the version from the 2.9.4 tag of play, and it compiles locally, but couldn't find the artifact in build jenkins https://github.com/playframework/playframework/blob/2.9.4/project/plugins.sbt#L16

Reply:
and when I request the url that it says it can't find, I'm able to download from artifactory - I will try rerunning the build

Reply:
trying doing the equiv play 2.9 bump in play-frontend-hmrc to see if that works, where we aren't directly using sbt-twirl but should be getting the same one via the play plugin

Reply:
https://github.com/hmrc/play-frontend-hmrc/pull/320

Reply:
Jo reran and it worked, so maybe a blip!